src/Controls/SubtitleListView.cs

```diff
@@ -50,23 +50,17 @@ public int GetColumnIndex(SubtitleColumn column)
 
         public override bool RightToLeftLayout
         {
-            get { return base.RightToLeftLayout; }
+            get
+            {
+                return base.RightToLeftLayout;
+            }

```

Comment:
` get => base.RightToLeftLayout;` would also work (vs2k17) didn't test it with 2k15